### PR TITLE
Document code cells

### PR DESCRIPTION
**Pull request by lgeiger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,12 +128,9 @@ moment, so please share with us the resolution to any rough spots that you find.
 
 ## Usage
 
-### Running code
-
-Hydrogen adds a command "Hydrogen: Run" to the command palette when you're in any text editor. Press ⌘-⇧-P to open the command palette and type "hydrogen" — it'll come up.
-
-The "Hydrogen: Run" command is bound to the keyboard shortcut ⌘-⌥-↩ by default.
+Hydrogen adds commands for running code to the command palette when you're in any text editor. Press ⌘-⇧-P to open the command palette and type "hydrogen" and they will come up.
```
Comment (n-riesco) [inline README.md]: Shorter sentences work better, e.g. *"Hydrogen provides a selection of commands for running code. Press [...]"*. There is no need to repeat what can be inferred from the context.

File: README.md [hunk 2]
```diff
@@ -149,12 +146,26 @@ It's easiest to see these interactions visually:
 
 <img src="http://g.recordit.co/4ViVmKtKAr.gif">
 
-If your code starts getting cluttered up with results, run "Hydrogen: Clear Results" to remove them all at once. You can also run this command with ⌘-⌥-⌫.
+**"Hydrogen: Run And Move Down"** will run the the code as described above and move the cursor to the next executable line.
+
+If your code starts getting cluttered up with results, run **"Hydrogen: Clear Results"** to remove them all at once. You can also run this command with ⌘-⌥-⌫.
+
+### "Hydrogen: Run Cell"
+A "code cell" is a block of lines to be executed at once. You can define them within inline comments. Hydrogen supports a
```
Comment (n-riesco) [inline README.md]: s/You can define them **within** inline comments/You can define them **using** inline comments/

**Comment by lgeiger** [comment]
I think we should ship another release after this is merged. What do you think?

**Comment by n-riesco** [comment]
On 25/07/16 15:27, Lukas Geiger wrote:
> I think we should ship another release after this is merged. What do you think?

Not yet, './hydrogen/connnection.json' is broken and the fix would touch the same code as #330.

**Pull request merged**